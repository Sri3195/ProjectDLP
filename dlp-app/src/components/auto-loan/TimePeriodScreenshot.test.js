import React from "react";
import {createRenderer} from 'react-test-renderer/shallow'

import { MemoryRouter } from "react-router-dom";
import TimePeriod from './TimePeriod'
//create a test renderer
const renderer=createRenderer()

describe("TimePeriod Screenshot test",()=>{
    test('renders correctly the time period  component',()=>{
        renderer.render(
            <MemoryRouter initialEntries={['/']}>
                <TimePeriod/>
          </MemoryRouter>
        );
        const result=renderer.getRenderOutput();
        expect(result).toMatchSnapshot();
    });
});