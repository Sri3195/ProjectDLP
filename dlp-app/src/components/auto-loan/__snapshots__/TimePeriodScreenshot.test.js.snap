// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TimePeriod Screenshot test renders correctly the time period  component 1`] = `
<Router
  location={
    Object {
      "hash": "",
      "key": "default",
      "pathname": "/",
      "search": "",
      "state": null,
    }
  }
  navigationType="POP"
  navigator={
    Object {
      "action": "POP",
      "createHref": [Function],
      "createURL": [Function],
      "encodeLocation": [Function],
      "go": [Function],
      "index": 0,
      "listen": [Function],
      "location": Object {
        "hash": "",
        "key": "default",
        "pathname": "/",
        "search": "",
        "state": null,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
>
  <TimePeriod />
</Router>
`;
